### PR TITLE
First Version

### PR DESCRIPTION
**Pull request by sbaerlocher**
### Added

- Initial release

### CODE REVIEW

**Review by FreeMinded — COMMENTED** [review]
Should the line 64 not say
- arillso.ca_certificates
as in line 12?

File: tasks/main.yml [hunk 1]
```diff
@@ -2,37 +2,45 @@
...
+  include_vars: '{{ loop_vars }}'
   with_first_found:
     - files:
-        - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-        - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
-        - "{{ ansible_distribution }}.yml"
-        - "{{ ansible_os_family }}.yml"
-        - "{{ ansible_system }}.yml"
-        - "defaults.yml"
+        - '{{ distribution }}-{{ distribution_verion }}.yml'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
        - '{{ distribution }}-{{ distribution_version }}.yml'
```

File: tasks/main.yml [hunk 2]
```diff
@@ -2,37 +2,45 @@
...
+        - '{{ ansible_system }}.yml'
+        - 'defaults.yml'
       paths:
-        - "vars"
+        - 'vars'
   loop_control:
     loop_var: loop_vars
+  vars:
+    distribution: '{{ ansible_distribution }}'
+    distribution_verion: '{{ ansible_distribution_version }}'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
    distribution_version: '{{ ansible_distribution_version }}'
```

File: tasks/main.yml [hunk 3]
```diff
@@ -2,37 +2,45 @@
...
+  include_tasks: '{{ loop_distribution }}'
   with_first_found:
     - files:
-        - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-        - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
-        - "{{ ansible_distribution }}.yml"
-        - "{{ ansible_os_family }}.yml"
-        - "{{ ansible_system }}.yml"
-        - "defaults.yml"
+        - '{{ distribution }}-{{ distribution_verion }}.yml'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
        - '{{ distribution }}-{{ distribution_version }}.yml'
```

File: tasks/main.yml [hunk 4]
```diff
@@ -2,37 +2,45 @@
...
+        - '{{ ansible_system }}.yml'
+        - 'defaults.yml'
       paths:
-        - "distribution"
+        - 'distribution'
   loop_control:
     loop_var: loop_distribution
+  vars:
+    distribution: '{{ ansible_distribution }}'
+    distribution_verion: '{{ ansible_distribution_version }}'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
    distribution_version: '{{ ansible_distribution_version }}'
```

**Review by mleutenegger — APPROVED** [review]

**Review by FreeMinded — APPROVED** [review]

**Pull request merged**